modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/TranslationManager.testcase

```diff
@@ -92,6 +92,62 @@ definition {
 		}
 	}
 
+	@description = "LPS-148142. Validates Manage Translations cancel add language by different ways."
+	@priority = "3"
+	test CancelAddLanguageByDifferentWays {
+		task ("Given manage translations modal") {
+			TranslationManagerPortlet.goToManageTranslationsModal(
+				key_currentLocale = "en-us",
+				key_title = "Admin");
+		}
+
+		task ("When add language in Manage Translations table list") {
+			ManageTranslations.addLanguage(key_locale = "zh-CN");
+		}
+
+		task ("Then cancel add a language with cancel button") {
+			Click(locator1 = "Button#CANCEL");
+		}
+
+		task ("Then view language not active in secltor menu") {
+			TranslationManagerPortlet.clickSelectTranslation(
+				key_currentLocale = "en-us",
+				key_title = "Admin");
+
+			AssertElementNotPresent(
+				key_locale = "zh-CN",
+				locator1 = "Translation#DROPDOWN_MENU_ITEM");
+
+			Click(locator1 = "//body");
```

Comment:
Is this step that clicking body necessary? Can this be done by clicking the select translation icon again? As the purpose of the step is not very clear.

Reply:
When I run the test locally, because the previous step is to assert, it is not closed after the menu is opened, which will affect the next operation.

Reply:
Thanks, I update it. I use click icon menu to replace it.